### PR TITLE
Add License to repo

### PR DESCRIPTION
**Pull request by irajdeep**
### All Submissions:

* [ ] Have you opened an Issue before filing this PR?
* [ ] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,3 +44,11 @@ $ helm repo add mongodb https://mongodb.github.io/helm-charts
 Please file issues before filing PRs. For PRs to be accepted, contributors must sign our [CLA](https://www.mongodb.com/legal/contributor-agreement).
 
 Reviewers, please ensure that the CLA has been signed by referring to [the contributors tool](https://contributors.corp.mongodb.com/) (internal link).
+
+
+
```
Comment (jamesbroadhead) [inline README.md:49]: Maybe a License heading here ?

File: APACHE2 [hunk 2]
```diff
@@ -0,0 +1,202 @@
+
```
[the remark this answers is not in the document]
Comment (priyolahiri) [reply APACHE2:1]: The file needs to be called LICENSE

File: README.md (hunk 1 above)
Comment (irajdeep) [reply README.md:49]: Addressed https://github.com/mongodb/helm-charts/pull/193/commits/ecce6f911ea0b75ca39251da087613a5a2e65cbe

File: APACHE2 (hunk 2 above)
[the remark this answers is not in the document]
Comment (irajdeep) [reply APACHE2]: Addressed https://github.com/mongodb/helm-charts/pull/193/commits/ecce6f911ea0b75ca39251da087613a5a2e65cbe

**Review by priyolahiri — APPROVED** [review]

**Pull request merged**